modules/nf-core/wgsim/main.nf

```diff
@@ -18,19 +18,34 @@ process WGSIM {
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
-
-    def WGSIM_VERSION = "0.3.1-r13" // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    def args    = task.ext.args ?: ''
+    def prefix  = task.ext.prefix ?: "${meta.id}"
+    def seed    = task.ext.seed ?: "${meta.seed}"
```

Comment:
I think the consensus is to not add `extra task.ext`,  you could just supply seed with ext.args right?

Reply:
makes sense - I basically copied what you did in the readsimulator pipeline to avoid breaking stuff but ext.args seems good?